mountinfo/mountinfo.go

```diff
@@ -16,15 +22,31 @@ func GetMountsFromReader(reader io.Reader, f FilterFunc) ([]*Info, error) {
 	return parseInfoFile(reader, f)
 }
 
-// Mounted determines if a specified mountpoint has been mounted.
-// On Linux it looks at /proc/self/mountinfo.
-func Mounted(mountpoint string) (bool, error) {
-	entries, err := GetMounts(SingleEntryFilter(mountpoint))
+// Mounted determines if a directory specified by path has been mounted.
```

Comment:
The old/current version of this function returns appropriate answers for mounted files as well as directories, right?  Are any of the existing usages of it sensitive to that distinction / would be broken by this change?

Reply:
Whoops, I entirely forgot about bind-mounted files scenario. Surely, this check needs to be removed.